spring-cloud-gcp-samples/spring-cloud-gcp-data-datastore-sample/src/main/java/com/example/DatastoreRepositoryExample.java

```diff
@@ -142,10 +142,15 @@ private void createRelationshipsInTransaction(Singer singerA, Singer singerB) {
 			return null;
 		});
 
-		System.out.println("Find by reference");
-		List<String> singers = this.singerRepository.findSingersByfirstBand(band3).stream().map(Singer::getFirstName)
+		System.out.println("Find by reference with query");
```

Comment:
remove debug system.outs (replace with logging if they are useful), but most likely replace with asserts.

Reply:
This is a sample app, and we use `System.out.println` for demonstration in our samples. The asserts are in the sample test.

Reply:
ah, ok -- makes sense.